src/content/app/genome-browser/services/genome-browser-service/types/zmenu.ts

```diff
@@ -85,6 +85,11 @@ export type ZmenuContentRegulationMetadata = {
   core_end: number; // end coordinate of the feature core
 };
 
+export type ZmenuContentOptionalMetadata = {
```

Comment:
I don't like the `OptionalMetadata` part of the name. First, it isn't clear what is optional — whether it's the metadata object itself or some of its keys. Secondly, according to the type, not even all the keys are optional.

`ZmenuContentMinimalMetadata`?